client/gutenberg/extensions/publicize/settings-button.jsx

```diff
@@ -45,10 +46,7 @@ class PublicizeSettingsButton extends Component {
 	render() {
 		return (
 			<div className="jetpack-publicize-add-connection-container">
-				<span
-					className="jetpack-publicize-add-icon dashicons-plus-alt"
-				>
-				</span>
+				<Dashicon icon="plus-alt" className="jetpack-publicize-add-icon" size={ 24 } />
```

Comment:
Does this render the corresponding `.jetpack-publicize-add-icon` styles irrelevant, at least the `font-family`?

Reply:
Removed the `font-family` per 1ac50a128e3427e4978cd3da09001e985f209cb4.